public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <!--<meta name="viewport" content="width=device-width, initial-scale=1" />-->
```

Comment:
I added a few separate tasks to fix the camera positioning and test on ios and android, since I couldn't come up with a quick fix here.

I don't know why the QR scanning doesn't work in Android emulator. I did customize some settings in my Android virtual device, maybe that is how it works or me?

Reply:
I tried testing in landscape and still was not able to get a QR to scan on Android. I tried emulating with a Pixel 2 so maybe it could be due to the emulating device or the customized settings that you configured.